### PR TITLE
Fix for being able to card a shunted AI.

### PR DESCRIPTION
**Pull request by gbasood**
Not tested, needs review. Looked at what shunt code did, saw mind transfer code, figured this would work

Also absolute pathing

### CODE REVIEW

File: code/game/objects/items/devices/aicard.dm [hunk 1]
```diff
@@ -13,6 +13,8 @@
 	attack(mob/living/silicon/ai/M as mob, mob/user as mob)
```
Comment (ghost) [inline code/game/objects/items/devices/aicard.dm]: Shouldn't that be properly casted?

Comment (gbasood) [inline code/game/objects/items/devices/aicard.dm]: What, the second arg? It's not important.

Comment (ghost) [inline code/game/objects/items/devices/aicard.dm]: I mean the attack() proc

Comment (gbasood) [inline code/game/objects/items/devices/aicard.dm]: Oh, you mean absolute pathed. Yes.

**Pull request merged**